LifeCycleAssessment_oM/Results/ProjectLifeCycleAssessmentResult.cs

```diff
@@ -0,0 +1,163 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Analytical.Results;
+using BH.oM.Base;
+using BH.oM.Quantities.Attributes;
+using System;
+using System.ComponentModel;
+
+namespace BH.oM.LifeCycleAssessment.Results
+{
+    [Description("A collection of simplified project results commonly used for database collection.")]
+    public partial class ProjectLifeCycleAssessmentResult : IResult, IImmutable
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("The proposed lifespan of the project scope after completion.")]
+        public virtual double BuildingLifespan { get; } = double.NaN;
+
+        [Description("Scope of the project typically new build or renovation.")]
+        public virtual string ConstructionScope { get; } = "";
+
+        [Description("Name of the individual compiling the results.")]
+        public virtual string ContactName { get; } = "";
+
+        [Description("Discipline scopes included in the LCA.")]
+        public virtual string ElementScope { get; } = "";
+
+        [Description("The primary structural system providing gravity support for the building.")]
+        public virtual string GravityStructuralMaterial { get; } = "";
+
+        [Description("The project total GlobalWarmingPotential (kgCO2e).")]
+        public virtual double GlobalWarmingPotential { get; } = double.NaN;
+
+        [Description("The project's GWP/m2 (kgCO2e/m2).")]
+        public virtual double GlobalWarmingPotentialPerArea { get; } = double.NaN;
+
+        [Description("The primary structural system providing lateral support for the building.")]
+        public virtual string LateralStructuralMaterial { get; } = "";
+
+        [Description("Phases included in the LCA concatenated (A1A2A3).")]
+        public virtual string Phases { get; } = "";
+
+        [Description("Typically a term utilised in BIM practices to clearly identify the scope of work being account for. Equivalents for LOD classifications can offen times be linked to design and construction phases common to the projects locale.")]
+        public virtual string LevelOfDevelopment { get; } = "";
+
+        [Description("The Project Area (m2) denotes the more precise project area which will allow assessment of kgCO2eq/m2 metrics.")]
+        [Area]
+        public virtual double ProjectArea { get; } = double.NaN;
```

Comment:
```suggestion
        public virtual double Area { get; } = double.NaN;
```

Would just call this `Area`